vsphere/datadog_checks/vsphere/utils.py

```diff
@@ -69,6 +92,11 @@ def is_metric_excluded_by_filters(metric_name, mor_type, metric_filters):
     return True
 
 
+def is_metric_included_by_filters(metric_name, mor_type, metric_filters):
```

Comment:
I think it confusing and probably hard to maintain to have both `is_metric_excluded_by_filters` and `is_metric_included_by_filters`, especially because they are not doing the same thing at all.

We should better state what functions are for, it's maybe a good idea to move the content of this function inside `should_collect_per_instance_values` as this behavior is different than other filters.

Reply:
I think that might be useful for later, but sure, I don't mind moving the content to `should_collect_per_instance_values`.